geopaparazzilibrary/src/eu/geopaparazzi/library/util/LibraryConstants.java

```diff
@@ -40,6 +40,17 @@
      */
     public final String SRID_MERCATOR_3857 = "3857"; //$NON-NLS-1$
     /**
+     * The epsg DHDN / Brandenburger Tor used by RasterLite2.
+     * - Images (Photos) imported to RasterLite2 will recieve the srid=-1
+     * - since this value cannot be transformed to Wsg84
+     * Since 2012, Spatialite contains historical 'DHDN / Soldner *' values that have been in use since 1879
+     * --> SELECT * FROM spatial_ref_sys WHERE auth_name = 'mj10777.de';
+     * I also 'sneaked' in an  imaginary Projection based on the Brandenburg Gate, Berlin, Germany at point 0,0
+     * - this (otherwise unused) Projection will be reserved to render Photo-Images
+     * -- since every Spatialite-Database with a full 'spatial_ref_sys',since 2012, contains this Projection.
+     */
+    public final String SRID_BRANDENBURGER_TOR_187999 = "187999"; //$NON-NLS-1$
+    /**
```

Comment:
Sorry, I answered these as a reply on the first day.
* but it seems it has to be answered here

The worked out values are in 3395 (WGS 84 / World Mercator)
* there was a reason for not using 3857 (it was worked out many years ago), but I don't remember the reason now
* 0,0 is the equator, where the width is the smallest

For the resolution if non-georeferenced Images (srid=-1)
* which always has a 0,0
   * it brings better results

Mapsforge does not support srid: -1 images, which can be stored in a RasterLite2 Database
* so the received wsg84 values must be transformed into something valid

The reason for its use as a const value, it because it is used in two functions